units/pod_allocator_test.go

```diff
@@ -223,6 +227,10 @@ func TestPodAllocatorJob(t *testing.T) {
 			}()
 			env.EvergreenSettings.PodLifecycle.MaxParallelPodRequests = 1
 			require.NoError(t, env.EvergreenSettings.PodLifecycle.Set())
+			env.EvergreenSettings.Providers.AWS.Pod.ECS.AllowedImages = []string{
```

Comment:
It seems weird to set the allowed images in a case which is testing something completely unrelated. Can we delete/move this? It seems like it's already set at the top of the test.

Reply:
I still have this question. If all of them need this setting, I would just specify the allowed images in the setup (i.e. above L53) and remove this spot, which sets it for just one test.

Reply:
I'm having some issues with this, I'll look into it soon

Reply:
moved it down

Reply:
(this should be good now, I had forgotten to push my last change)